nemo/collections/asr/models/ctc_bpe_models.py

```diff
@@ -121,26 +121,48 @@ def _setup_dataloader_from_config(self, config: Optional[Dict]):
         else:
             augmentor = None
 
-        dataset = AudioToBPEDataset(
-            manifest_filepath=config['manifest_filepath'],
-            tokenizer=self.tokenizer,
-            sample_rate=config['sample_rate'],
-            int_values=config.get('int_values', False),
-            augmentor=augmentor,
-            max_duration=config.get('max_duration', None),
-            min_duration=config.get('min_duration', None),
-            max_utts=config.get('max_utts', 0),
-            trim=config.get('trim_silence', True),
-            load_audio=config.get('load_audio', True),
-            add_misc=config.get('add_misc', False),
-        )
+        shuffle = config['shuffle']
+
+        # Instantiate tarred dataset loader or normal dataset loader
+        if 'is_tarred' in config and config.get('is_tarred', False):
```

Comment:
Why not just `if config.get('is_tarred', False)`?

Reply:
Let me try if that works (it probably does)

Reply:
Works !